### PR TITLE
Update limits-specifications-teams.md

### PR DESCRIPTION
**Pull request by kellybos**
Need to add limit about recording max size. Engineering provided information in customer escalation - https://portal.microsofticm.com/imp/v3/incidents/details/281130832/home.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/limits-specifications-teams.md [hunk 1]
```diff
@@ -118,6 +118,7 @@ Channel names also can't start with an underscore (_) or period (.), or end with
 |Number of people in a video or audio call from chat | 20 |
 |Max PowerPoint File Size | 2GB|
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:119]:
```suggestion
|Max PowerPoint File Size | 2 GB|
```

File: Teams/limits-specifications-teams.md [hunk 2]
```diff
@@ -118,6 +118,7 @@ Channel names also can't start with an underscore (_) or period (.), or end with
...
 |Teams keeps [meeting recordings](cloud-recording.md) that don't get uploaded to Microsoft Stream, available for local download | 20 days |
+| Meeting recording maximum length | 4 hours or 1.5 GB, when the limit is reached the recording will end and automatically restart
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:121]:
```suggestion
| Meeting recording maximum length | 4 hours or 1.5 GB. When this limit is reached, the recording will end and automatically restart.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kellybos, kindly commit the suggested changes. Many thanks.

File: Teams/limits-specifications-teams.md (hunk 1 above)
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:119]:
```suggestion
|Max PowerPoint File Size | 2 GB|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@kellybos - not related to this PR but could you kindly also commit the additional suggested change? Many thanks.

**Comment by kellybos** [comment]
@JohanFreelancer9 Sorry for the delay. Can you help get this document updated?

**Comment by JohanFreelancer9** [comment]
@scanum, could you please help to get this merged? Thanks.

**Comment by JohanFreelancer9** [comment]
@scanum?

**Comment by kellybos** [comment]
Any update on this request?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by AmitMohanty-G** [comment]
@JohanFreelancer9 Any update on this please? Customer is been waiting for this update for sometime now. I am MSFT please let me know if I can be any help to fast track it. Kelly is team-mate and she is on OOF.

**Comment by JohanFreelancer9** [comment]
@kellybos and @AmitMohanty-G. Unfortunately, I do not have the rights to merge this PR. @tiburd, could you possibly help to get this merged? Thank you so much.

**Comment by AmitMohanty-G** [comment]
@tiburd I am not sure why we are editing line 125. It is right now visible on the public facing TechNet. Weather that should be removed is a call with respective engineering PM. I am not authorized to approve that. I can only approve addition of line 119. Which @kellybos already did.

**Comment by AmitMohanty-G** [comment]
@tiburd patch9 with this "Meeting recording maximum length | 4 hours or 1.5 GB. When this limit is reached, the recording will end and automatically restart." looks good and correct to me.

**Comment by tiburd** [comment]
@AmitMohanty-G 
I'm sorry - I don't understand how your last comment can help resolve the file conflict. 

Again, for me to resolve the file conflict, I need to know whether the table row "Number of simultaneous calls | 4" needs to be removed:
image

Thank you

**Comment by AmitMohanty-G** [comment]
@tiburd 
I am not sure how to resolve the conflict. Can you please resolve the conflict. patch-9 branch are correct. please remove line number 125. "Number of simultaneous calls | 4":

**Pull request merged**